doc/adr/0002-use-changesets-for-publishing-and-versioning.md

```diff
@@ -0,0 +1,41 @@
+# 2: Use Changesets for Publishing Packages in the Monorepo
```

Comment:
Don't add ADRs on the same PR. ADRs should have their own space where we track and discuss them, to keep things simple. Ideally, before we implement stuff.

Reply:
Raised a seperate PR here: https://github.com/asyncapi/studio/pull/723